src/wymeditor/editor/undo-redo.js

```diff
@@ -23,6 +25,23 @@ WYMeditor.UndoRedo = function (wym) {
     // https://github.com/mightyiam/object-history
     undoRedo.history = new WYMeditor.EXTERNAL_MODULES
         .ObjectHistory(wym.getCurrentState());
+
+    wym.keyboard.combokeys.bind(
+        "mod+z",
+        function () {
+            wym.undoRedo.undo();
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    );
+    wym.keyboard.combokeys.bind(
+        ["shift+meta+z", "mod+y"],
```

Comment:
This isn't according to [their guidelines](https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/Keyboard.html#//apple_ref/doc/uid/20000957-CH84-SW3) but if you say the MSOffice for Mac uses command+y...